src/Tools/dotnet-trace/Profile.cs

```diff
@@ -22,7 +22,7 @@ public Profile(string name, IEnumerable<EventPipeProvider> providers, string des
 
         public string Description { get; }
 
-        public bool Rundown { get; set;  }
```

Comment:
We should change this in the commandline options, have the default be true, and not pass down a nullable.

Reply:
we still need to take care of the case that if the user specified both profile and rundown on the commandline, the specified rundown should override what the profile specifies.